### ISSUE TITLE
Upgrading from 0.5 to 0.5.1 - all channels active=false

### ISSUE THREAD

**Issue by bitcoinshirt** [lightningnetwork/lnd#2292]
### Background

Upgraded a pruned noseedbackup=1 LND 0.5.0 beta to 0.5.1, after upgrading, channels are inactive, but not force-closed.

### Environment
* Pruned BTCPay Node
* noseedbackup=1
* 0.5.1

I'm not sure if this is relevant but:
* I had some problems earlier with minrelaytxfee=0 more info in  #2052 #2059

### Steps to reproduce

Here are the logs.

```
BTC: Is fully synched
Node synched
Attempting automatic RPC configuration to bitcoind
Automatically obtained bitcoind's RPC credentials
2018-12-07 12:15:00.902 [INF] LTND: Version: 0.5.1-beta commit=basedon-v0.5.1-beta-dirty, build=production, logging=default
2018-12-07 12:15:00.902 [INF] LTND: Active chain: Bitcoin (network=mainnet)
2018-12-07 12:15:00.928 [INF] CHDB: Checking for schema update: latest_version=7, db_version=6
2018-12-07 12:15:00.928 [INF] CHDB: Performing database schema migration
2018-12-07 12:15:00.928 [INF] CHDB: Applying migration #7
2018-12-07 12:15:00.928 [INF] CHDB: Migrating to new closed channel format...
2018-12-07 12:15:00.951 [INF] CHDB: Migration to new closed channel format complete!
2018-12-07 12:15:01.118 [INF] LTND: Primary chain is set to: bitcoin
2018-12-07 12:15:01.151 [INF] LTND: Initializing bitcoind backed fee estimator
2018-12-07 12:15:01.152 [INF] LNWL: Started listening for bitcoind block notifications via ZMQ on tcp://bitcoind:28332
2018-12-07 12:15:01.152 [INF] LNWL: Started listening for bitcoind transaction notifications via ZMQ on tcp://bitcoind:28333
2018-12-07 12:15:01.161 [INF] LNWL: Applying wallet transaction manager migration #2
2018-12-07 12:15:01.161 [INF] LNWL: Dropping wallet transaction history
2018-12-07 12:15:01.196 [INF] LNWL: Applying wallet address manager migration #6
2018-12-07 12:15:01.200 [INF] LNWL: Setting the wallet's birthday block from timestamp=0000-12-30 00:00:00 +0000 UTC
2018-12-07 12:15:01.229 [INF] LNWL: Estimated birthday block from timestamp=0000-12-30 00:00:00 +0000 UTC: height=0, hash=000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f
2018-12-07 12:15:01.229 [INF] LNWL: Applying wallet address manager migration #7
2018-12-07 12:15:14.251 [INF] LNWL: Opened wallet
2018-12-07 12:15:17.063 [INF] LNWL: The wallet has been unlocked without a time limit
2018-12-07 12:15:17.129 [INF] LTND: LightningWallet opened
2018-12-07 12:15:17.239 [INF] HSWC: Restoring in-memory circuit state from disk
2018-12-07 12:15:17.260 [INF] HSWC: Payment circuits loaded: num_pending=1, num_open=0
2018-12-07 12:15:17.689 [INF] LNWL: Catching up block hashes to height 552894, this will take a while...
2018-12-07 12:15:17.893 [INF] LNWL: Caught up to height 0
2018-12-07 12:15:17.992 [INF] HSWC: Trimming open circuits for chan_id=546257:417:0, start_htlc_id=0
2018-12-07 12:15:17.992 [INF] HSWC: Trimming open circuits for chan_id=550358:747:0, start_htlc_id=0
2018-12-07 12:15:17.992 [INF] HSWC: Trimming open circuits for chan_id=551737:2689:1, start_htlc_id=1
2018-12-07 12:15:17.992 [INF] HSWC: Trimming open circuits for chan_id=552752:1338:0, start_htlc_id=0
2018-12-07 12:15:17.993 [INF] HSWC: Trimming open circuits for chan_id=550422:1298:0, start_htlc_id=0
2018-12-07 12:15:17.993 [INF] HSWC: Trimming open circuits for chan_id=544608:1026:0, start_htlc_id=94
2018-12-07 12:15:17.993 [INF] HSWC: Trimming open circuits for chan_id=550051:1203:1, start_htlc_id=104
2018-12-07 12:15:17.993 [INF] HSWC: Trimming open circuits for chan_id=550136:2008:1, start_htlc_id=7
2018-12-07 12:15:17.993 [INF] HSWC: Trimming open circuits for chan_id=545053:1118:0, start_htlc_id=0
2018-12-07 12:15:17.993 [INF] HSWC: Trimming open circuits for chan_id=541347:986:1, start_htlc_id=0
2018-12-07 12:15:17.994 [INF] HSWC: Trimming open circuits for chan_id=541266:778:0, start_htlc_id=15
2018-12-07 12:15:17.994 [INF] HSWC: Trimming open circuits for chan_id=541552:2103:1, start_htlc_id=7
2018-12-07 12:15:17.994 [INF] HSWC: Trimming open circuits for chan_id=552010:552:0, start_htlc_id=0
2018-12-07 12:15:17.994 [INF] HSWC: Trimming open circuits for chan_id=543691:1106:0, start_htlc_id=0
2018-12-07 12:15:17.994 [INF] HSWC: Trimming open circuits for chan_id=550634:2242:1, start_htlc_id=0
2018-12-07 12:15:17.995 [INF] HSWC: Trimming open circuits for chan_id=541980:577:0, start_htlc_id=0
2018-12-07 12:15:17.995 [INF] HSWC: Trimming open circuits for chan_id=552071:898:1, start_htlc_id=2
2018-12-07 12:15:17.995 [INF] HSWC: Trimming open circuits for chan_id=552088:1911:0, start_htlc_id=0
2018-12-07 12:15:17.995 [INF] HSWC: Trimming open circuits for chan_id=552062:2350:0, start_htlc_id=0
2018-12-07 12:15:17.995 [INF] HSWC: Trimming open circuits for chan_id=550436:1227:0, start_htlc_id=0
2018-12-07 12:15:17.995 [INF] HSWC: Trimming open circuits for chan_id=549531:1932:1, start_htlc_id=0
2018-12-07 12:15:17.996 [INF] HSWC: Trimming open circuits for chan_id=544853:1016:0, start_htlc_id=5
2018-12-07 12:15:17.996 [INF] HSWC: Trimming open circuits for chan_id=551980:1328:1, start_htlc_id=0
2018-12-07 12:15:17.996 [INF] HSWC: Trimming open circuits for chan_id=550333:1384:0, start_htlc_id=11
2018-12-07 12:15:17.996 [INF] HSWC: Trimming open circuits for chan_id=541394:2225:0, start_htlc_id=1
2018-12-07 12:15:17.996 [INF] HSWC: Trimming open circuits for chan_id=541271:1321:0, start_htlc_id=0
2018-12-07 12:15:17.996 [INF] HSWC: Trimming open circuits for chan_id=549712:372:0, start_htlc_id=0
2018-12-07 12:15:17.996 [INF] HSWC: Trimming open circuits for chan_id=541476:2227:0, start_htlc_id=0
2018-12-07 12:15:17.996 [INF] HSWC: Trimming open circuits for chan_id=542095:2159:0, start_htlc_id=0
2018-12-07 12:15:17.997 [INF] HSWC: Trimming open circuits for chan_id=544926:1000:0, start_htlc_id=0
2018-12-07 12:15:17.997 [INF] HSWC: Trimming open circuits for chan_id=550242:321:0, start_htlc_id=0
2018-12-07 12:15:17.997 [INF] HSWC: Trimming open circuits for chan_id=549171:1852:0, start_htlc_id=0
2018-12-07 12:15:17.997 [INF] HSWC: Trimming open circuits for chan_id=545484:1315:0, start_htlc_id=0
2018-12-07 12:15:17.997 [INF] HSWC: Trimming open circuits for chan_id=552207:1514:1, start_htlc_id=0
2018-12-07 12:15:17.997 [INF] HSWC: Trimming open circuits for chan_id=550442:2038:0, start_htlc_id=0
2018-12-07 12:15:17.997 [INF] HSWC: Trimming open circuits for chan_id=552716:1087:1, start_htlc_id=0
2018-12-07 12:15:17.998 [INF] HSWC: Trimming open circuits for chan_id=541259:1547:0, start_htlc_id=59
2018-12-07 12:15:17.998 [INF] HSWC: Trimming open circuits for chan_id=543937:2163:1, start_htlc_id=0
2018-12-07 12:15:17.998 [INF] HSWC: Trimming open circuits for chan_id=550446:949:1, start_htlc_id=0
2018-12-07 12:15:17.998 [INF] HSWC: Trimming open circuits for chan_id=546069:1632:0, start_htlc_id=0
2018-12-07 12:15:17.998 [INF] HSWC: Trimming open circuits for chan_id=550175:413:1, start_htlc_id=41
2018-12-07 12:15:17.998 [INF] HSWC: Trimming open circuits for chan_id=551527:464:0, start_htlc_id=0
2018-12-07 12:15:17.999 [INF] HSWC: Trimming open circuits for chan_id=551557:1315:1, start_htlc_id=0
2018-12-07 12:15:17.999 [INF] HSWC: Trimming open circuits for chan_id=549094:448:1, start_htlc_id=7
2018-12-07 12:15:17.999 [INF] HSWC: Trimming open circuits for chan_id=547285:598:0, start_htlc_id=0
2018-12-07 12:15:18.026 [INF] RPCS: RPC server listening on 127.0.0.1:10008
2018-12-07 12:15:18.027 [INF] RPCS: RPC server listening on [::]:10009
2018-12-07 12:15:18.029 [INF] RPCS: gRPC proxy started at [::]:8080
2018-12-07 12:15:18.047 [INF] LTND: Waiting for chain backend to finish sync, start_height=552894
2018-12-07 12:15:41.399 [INF] LNWL: Caught up to height 10000
2018-12-07 12:16:04.147 [INF] LNWL: Caught up to height 20000
2018-12-07 12:16:26.918 [INF] LNWL: Caught up to height 30000
2018-12-07 12:16:50.166 [INF] LNWL: Caught up to height 40000
2018-12-07 12:17:14.546 [INF] LNWL: Caught up to height 50000
2018-12-07 12:17:40.113 [INF] LNWL: Caught up to height 60000
2018-12-07 12:18:04.496 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:18:04.719 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:18:10.746 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:18:10.972 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:18:14.097 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:18:14.300 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:18:14.512 [INF] LNWL: Caught up to height 70000
2018-12-07 12:18:17.352 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:18:17.396 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:18:27.215 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:18:27.239 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:18:28.917 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:18:28.939 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:18:35.066 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:18:35.216 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:18:36.510 [INF] LNWL: Caught up to height 80000
2018-12-07 12:18:52.877 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:18:53.033 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:18:59.467 [INF] LNWL: Caught up to height 90000
2018-12-07 12:19:01.708 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:19:01.733 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:19:14.414 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:19:14.566 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:19:16.087 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:19:16.230 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:19:22.260 [INF] LNWL: Caught up to height 100000
2018-12-07 12:19:32.035 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:19:32.179 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:19:45.743 [INF] LNWL: Caught up to height 110000
2018-12-07 12:20:09.346 [INF] LNWL: Caught up to height 120000
2018-12-07 12:20:11.863 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:20:11.901 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:20:33.061 [INF] LNWL: Caught up to height 130000
2018-12-07 12:20:42.950 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:20:43.094 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:20:56.101 [INF] LNWL: Caught up to height 140000
2018-12-07 12:21:18.098 [INF] LNWL: Caught up to height 150000
2018-12-07 12:21:40.039 [INF] LNWL: Caught up to height 160000
2018-12-07 12:22:04.372 [INF] LNWL: Caught up to height 170000
2018-12-07 12:22:26.876 [INF] LNWL: Caught up to height 180000
2018-12-07 12:22:29.377 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:22:29.519 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:22:30.905 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:22:31.054 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:22:32.409 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:22:32.566 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:22:48.332 [INF] LNWL: Caught up to height 190000
2018-12-07 12:23:11.668 [INF] LNWL: Caught up to height 200000
2018-12-07 12:23:33.131 [INF] LNWL: Caught up to height 210000
2018-12-07 12:23:54.940 [INF] LNWL: Caught up to height 220000
2018-12-07 12:24:17.192 [INF] LNWL: Caught up to height 230000
2018-12-07 12:24:38.662 [INF] LNWL: Caught up to height 240000
2018-12-07 12:25:00.231 [INF] LNWL: Caught up to height 250000
2018-12-07 12:25:23.522 [INF] LNWL: Caught up to height 260000
2018-12-07 12:25:44.691 [INF] LNWL: Caught up to height 270000
2018-12-07 12:26:07.667 [INF] LNWL: Caught up to height 280000
2018-12-07 12:26:28.895 [INF] LNWL: Caught up to height 290000
2018-12-07 12:26:51.085 [INF] LNWL: Caught up to height 300000
2018-12-07 12:27:13.515 [INF] LNWL: Caught up to height 310000
2018-12-07 12:27:35.093 [INF] LNWL: Caught up to height 320000
2018-12-07 12:27:48.152 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:27:48.246 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:27:55.227 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:27:55.385 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:27:58.145 [INF] LNWL: Caught up to height 330000
2018-12-07 12:28:18.438 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:28:18.581 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:28:21.951 [INF] LNWL: Caught up to height 340000
2018-12-07 12:28:45.271 [INF] LNWL: Caught up to height 350000
2018-12-07 12:29:08.583 [INF] LNWL: Caught up to height 360000
2018-12-07 12:29:31.821 [INF] LNWL: Caught up to height 370000
2018-12-07 12:29:54.978 [INF] LNWL: Caught up to height 380000
2018-12-07 12:30:20.105 [INF] LNWL: Caught up to height 390000
2018-12-07 12:30:42.068 [INF] LNWL: Caught up to height 400000
2018-12-07 12:30:48.240 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:30:48.261 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:31:05.507 [INF] LNWL: Caught up to height 410000
2018-12-07 12:31:27.870 [INF] LNWL: Caught up to height 420000
2018-12-07 12:31:30.062 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:31:30.090 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:31:40.742 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:31:40.768 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:31:51.543 [INF] LNWL: Caught up to height 430000
2018-12-07 12:32:14.966 [INF] LNWL: Caught up to height 440000
2018-12-07 12:32:39.624 [INF] LNWL: Caught up to height 450000
2018-12-07 12:33:03.104 [INF] LNWL: Caught up to height 460000
2018-12-07 12:33:24.614 [INF] LNWL: Caught up to height 470000
2018-12-07 12:33:46.094 [INF] LNWL: Caught up to height 480000
2018-12-07 12:34:09.916 [INF] LNWL: Caught up to height 490000
2018-12-07 12:34:26.056 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:34:33.395 [INF] LNWL: Caught up to height 500000
2018-12-07 12:34:57.108 [INF] LNWL: Caught up to height 510000
2018-12-07 12:35:21.524 [INF] LNWL: Caught up to height 520000
2018-12-07 12:35:45.137 [INF] LNWL: Caught up to height 530000
2018-12-07 12:36:08.053 [INF] LNWL: Caught up to height 540000
2018-12-07 12:36:30.437 [INF] LNWL: Caught up to height 550000
2018-12-07 12:36:37.447 [INF] LNWL: Done catching up block hashes
2018-12-07 12:36:37.582 [INF] LNWL: Started rescan from block 000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f (height 0) for 505 addresses
2018-12-07 12:36:37.589 [INF] LNWL: Starting rescan from block 000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f
2018-12-07 12:36:37.646 [INF] LNWL: Rescan finished at 0 (000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f)
2018-12-07 12:36:37.646 [ERR] LNWL: Unable to complete chain rescan: -1: Block not available (pruned data)
2018-12-07 12:36:37.646 [INF] LNWL: Catching up block hashes to height 0, this might take a while
2018-12-07 12:36:37.652 [INF] LNWL: Done catching up block hashes
2018-12-07 12:36:37.652 [INF] LNWL: Finished rescan for 505 addresses (synced to block 000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f, height 0)
2018-12-07 12:38:09.192 [ERR] LNWL: Failed to process consensus server notification (name: `blockconnected`, detail: `failed to store sync information 0000000000000000002147d9ccd0d19c3f00eeb5d25c380be4bc8a1544a7a1b5: failed to fetch block hash for height 552895: block not found`)
2018-12-07 12:39:11.702 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:39:11.735 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:41:44.859 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:41:44.881 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:41:47.948 [ERR] LNWL: Failed to process consensus server notification (name: `blockconnected`, detail: `failed to store sync information 0000000000000000001c9b511ae5ed3cf31b6217f641662cfe9c243d9d90e233: failed to fetch block hash for height 552896: block not found`)
2018-12-07 12:44:11.515 [ERR] LNWL: Failed to process consensus server notification (name: `blockconnected`, detail: `failed to store sync information 0000000000000000001cf17872958b2e3e077432d00ec5716bbe1e0abfa81178: failed to fetch block hash for height 552897: block not found`)
2018-12-07 12:45:09.162 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:45:09.193 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:45:11.149 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:45:11.248 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:45:37.215 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:45:37.282 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:47:24.838 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:47:24.983 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:47:36.729 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:47:36.790 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:48:13.890 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:48:22.955 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:48:22.978 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:52:07.116 [ERR] LNWL: Failed to process consensus server notification (name: `blockconnected`, detail: `failed to store sync information 000000000000000000182d7a720bc27cb0829ab2d28bb827576c66cf47fff19d: failed to fetch block hash for height 552898: block not found`)
2018-12-07 12:56:48.914 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:56:48.944 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:56:51.894 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:56:51.919 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:56:59.846 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:56:59.899 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:58:30.056 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 12:58:30.185 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 12:58:55.290 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 13:11:34.023 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 13:11:34.098 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
2018-12-07 13:11:47.548 [INF] RPCS: [listchannels] fetched 45 channels from DB
2018-12-07 13:11:47.574 [INF] UTXN: NurseryReport: building nursery report for channel 4c4192dd1be88b5da02f36ea4b1ab4949043579a01379fb7d90776f9230e8ce2:1
```

### Expected behaviour

I expected that funds in on-chain wallet would be wiped, so I moved them, I did not expect channels to become inactive.

### Actual behaviour

Channels entered inactive state. Good thing is they aren't force-closed. The only channel that was left active is channel that was in pending state for 2 months.

### Additional information
On Slack, user Moli, suggested interesting solution to my problem
> .lnd to a full no-pruned bitcoind and rescan it until it's ok then put it back to btcpay, it might work but i have no idea if it will

So I'd like to ask if there are other solutions I can attempt prior to this?
Thanks.

**Comment by Roasbeef** [comment]
So the issue is this: 
```
2018-12-07 12:15:01.200 [INF] LNWL: Setting the wallet's birthday block from timestamp=0000-12-30 00:00:00 +0000 UTC
2018-12-07 12:15:01.229 [INF] LNWL: Estimated birthday block from timestamp=0000-12-30 00:00:00 +0000 UTC: height=0, hash=000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f
2018-12-07 12:15:01.229 [INF] LNWL: Applying wallet address manager migration #7
```

BTCPay made nodes without a seed, as a result, we don't know what the wallet's birthday is and are forced to rescan from genesis. Bitcoind doesn't have the blocks we need, so the rescan will fail: 
```
2018-12-07 12:36:37.646 [ERR] LNWL: Unable to complete chain rescan: -1: Block not available (pruned data)
```

**Comment by Roasbeef** [comment]
> On Slack, user Moli, suggested interesting solution to my problem
That should work, as long as the node has all blocks in the chain. Note though, since your wallet doesn't have a birthday, that it'll take a _very_ long time since it needs to fetch and scan all blocks in the chain.

**Comment by bitcoinshirt** [comment]
Thanks @Roasbeef for the help. I'll try to figure out how to do that in BTCPay and report back in few days. Appreciate the assistance as always.

**Comment by bitcoinshirt** [comment]
Okay guys, it's back online! It was easier than expected.

I suspected the issue I was having is related to `minrelaytxfee=0` more info in #2052 #2059
Upon initial start, it appears that the node was stuck and channels inactive.

I did not know what else to do, even though **I initially applied** `minrelaytxfee=0`  as soon as it started, I applied it again, and it appears it's all back now. 
Channels are SAFU. (maybe 1 force closed or something) Funds are SAFU.

However I do think there may be a bug of some and will be sending full logs shortly. Let's not close this issue before someone makes sure it's not a bug.

**Comment by Roasbeef** [comment]
> interesting, my own node continued working fine after the update. Even if I pruned after the birthday.

It'll "work", in that your channels will be there, but the issue is that the rescan cannot proceed. Also it depends on how old your node is, super old nodes don't have a wallet birthday is they were created with no seed backup (like the OP), newer nodes will have one however, which speeds up the rescan dramatically.

**Comment by Roasbeef** [comment]
> However I do think there may be a bug of some and will be sending full logs shortly

There's no bug, your channels will be there, and you'll be able to send/recv coins. However, the rescan cannot proceed without all the blocks since the wallet is starting from genesis to complete the rescan.

**Comment by bitcoinshirt** [comment]
That's how it all happened @Roasbeef thank you. As you said, I noticed that my on-chain wallet doesn't have all the funds (but I expected that). 

I will look into finding a node to help with a re-scan.

Thanks for your help. I'm closing this for now. For me, it's important that customers can send their payments.

**Comment by BobleChinois** [comment]
Hi everyone, I encountered the same issue than OP, when I upgraded my BTCPay server all my channels appeared to be inactive. I did a full rescan with a non pruned full node, most of my channels were back after the rescan, but then I turned LND off and copied the content of .lnd back to my BTCPay only to find out that LND was trying to do a full rescan again.

Worse, I fired up LND on my (non pruned) full node again, and it started the full rescan like nothing happened.

What am I missing here?

**Comment by BobleChinois** [comment]
Some update: I did a full rescan once again, I see the force closed channels with some local balance in pending state, so I'll wait for it to complete and see if it comes back. 
On the bright side, I found a tx from a couple of weeks ago, it seems that I withdraw the content of my wallet and totally forgot I did, so it seems there's no loss at least on this side. 
Besides taking back what I had on channels, my main issue now is how can I run an "after rescan" LND on my BTCPay server. Is it normal behaviour for LND to need a full rescan from wallet's birthday every time you restart it?
